java/daikon/dcomp/DCInstrument.java

```diff
@@ -427,8 +427,10 @@ public JavaClass instrument() {
         stack_trace = Thread.currentThread().getStackTrace();
         // [0] is getStackTrace
         for (int i = 1; i < stack_trace.length; i++) {
-          if (stack_trace[i].getClassName().contains("JUnitCommandLineParseResult")
-              && stack_trace[i].getMethodName().equals("parse")) {
+          // debug code
```

Comment:
Minor: An alternative to the `// debug code` comments would be to add a new `static final boolean debug` variable.  Then, a programmer can turn all the debugging on or off by editing its assignment, and the `// debug code` comments would not be necessary.